sourcemash/frontend/templates/category.tpl

```diff
@@ -7,11 +7,25 @@
 
   <div class="loading center-align"></div>
 
-  <ul id="items" class="list-group row">
-    <% items.forEach(function(item) { %>
-        <div id="item-<%= item.get('id') %>" class="item-card"></div>
-    <% }); %>
-  </ul>
+  <div id="items" class="list-group row">
+    <div class="col m6 hide-on-small-and-down">
```

Comment:
No, but might be a rap lyric...

Small is just 600px, small-and-down sets max-width of 600px.